src/netCDF4/_netCDF4.pyx

```diff
@@ -2757,6 +2758,17 @@ is the number of variable dimensions."""
             group = self
         else:
             group = self.createGroup(dirname)
+        # if dimensions is a single string or Dimension instance, 
+        # convert to a tuple.
+        # This prevents a common error that occurs when
+        # dimensions = 'lat' instead of ('lat',)
+        if type(dimensions) == str or type(dimensions) == bytes or\
```

Comment:
why not just `isinstance(dimensions, (str, bytes, Dimension))`? type forces the type but not sub-classes?

Reply:
done